addons/mail/static/src/discuss/core/common/message_model_patch.js

```diff
@@ -79,5 +82,12 @@ const messagePatch = {
     showSeenIndicator(thread) {
         return this.isSelfAuthored && thread?.hasSeenFeature;
     },
+    async fetchDescendants() {
+        const { data, descendants } = await rpc("/discuss/message/descendants", {
+            message_id: this.id,
+        });
+        this.store.insert(data);
+        this.descendants = descendants;
+    },
```

Comment:
We might want to move it to /mail/data instead

Reply:
It doesn't need to be sent separately if it's written on a field on the model just like any other field.